GetDKAN/dkan_feedback
# Show feedback user's chosen name

Connects #18 

## Description

The "Add Feedback" form should ask for a user-supplied name, and then show that field on the published feedback (currently all feedback is "by Anonymous", which doesn't look great)

## Steps to test:

- To get this working in your local add the following piece of code in `src/make/drupal.make`:

```
dkan_feedback:
    download:
      type: git 
      url: https://github.com/NuCivic/dkan_feedback.git
      branch: 18-feedback-author
    type: module
```

Then run `dktl make` and enable dkan_feedback.

- Create a new feedback node as an anonymous user, you should see a field `Contact Name` right before the `Email` field.
- Create a new feedback node as an authenticated user, you should see a field `Contact Name` right before the `Email` field, it should be pre-filled with the username.
- On the feedback node page, you should see the `Contact Name` on the feedback group table.
- When the feedback nodes have been published, if you go to `/feedback` you should see the text `posted by ` and the name the user entered in the field `Contact Name`. If there were already feedback nodes and the `Contact Name` is empty for them, then they should be shown as `posted by Anonymous`.
